### DISCUSSION TITLE
when I finetune CLIP_ViT_L_14 model , Logit Scale is decrease from 100.0 to 95. and keep going , is right?

### DISCUSSION THREAD

**Discussion by Johnson-yue** [mlfoundations/open_clip#953] [Q&A]
Hi, everybody, when I finetune ViT_L_14 model  with my own dataset, the log is :
1) Logit Scale is decrease from 100.0  to 95 (at epoch 1) and keep going.
2) loss is about 0.0xxx 
3) | Eval Epoch: 1 
image_to_text_mean_rank: 2.3887	
image_to_text_median_rank: 2.0000	
image_to_text_R@1: 0.4132	
image_to_text_R@5: 0.9666	
image_to_text_R@10: 0.9917	
text_to_image_mean_rank: 2.6540	
text_to_image_median_rank: 2.0000	
text_to_image_R@1: 0.4230	
text_to_image_R@5: 0.9745	
text_to_image_R@10: 0.9935	
clip_val_loss: 0.0076
4)  train log:
<img width="841" alt="image" src="https://github.com/user-attachments/assets/3d0c49f4-a97e-4221-80eb-3d861853a95e">
5) val log :  
<img width="1660" alt="image" src="https://github.com/user-attachments/assets/6233681e-4f88-4ba8-b551-50684cbafad7">
should I  shut it down ?
